dashboard/src/modules/components/EmptyPageComponent/NoMatchingPage.jsx

```diff
@@ -0,0 +1,35 @@
+import "./index.less";
+
+import { Button } from "@patternfly/react-core";
+import React from "react";
+import logo from "assets/images/404-Error-Not-found.png";
```

Comment:
Perhaps call this "page_not_found".

Reply:
Yes, we need a hyphen that is shaped like a comma.  😆  It's not "Hey, we didn't find the 404-Error", it's "Hey it's the not-found error, 404".  🤣

Seriously, though, Peter's is a good solution:  this is a case where "less is more" -- having "404" _and_ "error" _and_ "not found" all in the name makes things less clear with more room for silliness.

Reply:
Renamed the image :grinning: